signing/src/main/java/tech/pegasys/web3signer/signing/config/SignerLoader.java

```diff
@@ -120,7 +120,12 @@ private MappedResults<SigningMetadata> convertConfigFileContent(
   }
 
   private static String calculateTimeTaken(final Instant start) {
-    return DurationFormatUtils.formatDurationHMS(Duration.between(start, Instant.now()).toMillis());
+    final Instant now = Instant.now();
+    final long timeTaken = Duration.between(start, now).toMillis();
+    if (timeTaken < 0) {
+      LOG.warn("System Clock returned time in past. Start: {}, Now: {}.", start, now);
+    }
+    return DurationFormatUtils.formatDurationHMS(Math.abs(timeTaken));
```

Comment:
Think this could still throw the error when `timeTaken < 0`?

Reply:
I see the Math.abs will avoid the error, but it will also mean we report inaccurate time taken so better not to report anything in this case

Reply:
Agree ABS doesn't seem like the right solution. We should just not return anything in that case. So maybe just the return type to be Optional<String> and return the empty in the invalid case.

Reply:
addressed.